runtime.md

```diff
@@ -129,18 +129,19 @@ This operation MUST run the user-specified program as specified by [`process`](c
 This operation MUST generate an error if `process` was not set.
 
 ### <a name="runtimeKill" />Kill
-`kill <container-id> <signal>`
+`kill <container-id> [-a,--all] <signal>`
 
 This operation MUST [generate an error](#errors) if it is not provided the container ID.
-Attempting to send a signal to a container that is neither [`created` nor `running`](#state) MUST have no effect on the container and MUST [generate an error](#errors).
-This operation MUST send the specified signal to the container process.
+This operation MUST send the specified signal to the container process(without `-a`) or to all processes in the container(with `-a`).
+Without `-a`, attempting to send a signal to a container that is neither [`created` nor `running`](#state) MUST have no effect on the container and MUST [generate an error](#errors).
 
 ### <a name="runtimeDelete" />Delete
-`delete <container-id>`
+`delete [-f,--force] <container-id>`
 
 This operation MUST [generate an error](#errors) if it is not provided the container ID.
-Attempting to `delete` a container that is not [`stopped`](#state) MUST have no effect on the container and MUST [generate an error](#errors).
-Deleting a container MUST delete the resources that were created during the `create` step.
+Without `-f`, attempting to `delete` a container that is not [`stopped`](#state) MUST have no effect on the container and MUST [generate an error](#errors).
+With `-f`, runtime should try its best to delete the container no matter what the state is.
+Deleting a container MUST delete the resources that were created during the `create` step, specially, if the container uses `shared pid namespace`, all processes inside the container's cgroup should also be killed first.
```

Comment:
I'd rephrase it as "if the container doesn't use a PID namespace`

Reply:
Sorry, I haven't understood your meaning yet.
You mean change "if the container uses `shared pid namespace`" to "if the container doesn't use a PID namespace`"?

Reply:
> You mean change "if the container uses `shared pid namespace`" to "if the container doesn't use a PID namespace`"?

If this is your wanted change, I think it can't include all the situations, because in here `shared pid namespace` means:
1. Doesn't use a PID namespace (Using the host pid namespace);
2. Shares a PID namespaces with other container's processes(Using a specified process' pid namespace, the process may be not in this container).

So maybe we should keep "if the container uses `shared pid namespace`"?
@giuseppe PTAL, thanks very much.

Reply:
@lifubang I think the best wording would be
```
s/if the container uses `shared pid namespace`/if the container does not use its own private PID namespace/
```
or somesuch.